qiita_db/metadata_template.py

```diff
@@ -2289,6 +2290,15 @@ def load_template_to_dataframe(fn, strip_whitespace=True):
                                          " cannot be cast into a %s "
                                          "type." % (n, c_dtype))
 
+    # get and clean the required columns
```

Comment:
Although this works, this does not enforce the requirements above i.e. latitude/longitude being a number, sample_name always being a string, etc. The normalization to lowercase has to be done before we parse through pandas, otherwise we cannot guarantee these requirements are met.